benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -267,23 +267,16 @@ export const constructHashURL = (url, params = {}) => {
 };
 
 /**
- * Sets or updates the search parameters in the URL hash of the current page. All the existing search parameters will not be disturbed. Also accepts a history object to update the URL hash without reloading the page.
+ * Sets or updates the search parameters in the URL hash of the current page. All the existing search parameters will not be disturbed.
  * It can also be used to remove a parameter from the URL by setting it's value to undefined.
  *
  * @param {Object} params - The parameters to be set or updated in the URL hash
- * @param {Object} [history=null] - The history object to use for updating the URL hash
  * @returns {void}
  */
-const setURLParameter = (params = {}, history = null) => {
-  const { newUrl, queryString } = constructHashURL(
-    document.location.href,
-    params,
-  );
-  if (history && history.push) {
-    history.push(queryString);
-    return;
-  }
-  document.location.href = newUrl;
+const setURLParameter = (params = {}, history = undefined) => {
```

Comment:
Remove `history`?

Reply:
Yes! It is not used in the codebase now.